### DISCUSSION TITLE
I'm not dead

### DISCUSSION THREAD

**Discussion by SlashScreen** [SlashScreen/skelerealms#24] [Announcements]
Hi there everyone! I wanted to say that this project has not been abandoned. I've been busy with school, health issues, and the new WoW expansion (sorry). Lately I have been working on smaller projects, and making a much more flexible GOAP system as a [separate plugin](https://github.com/SlashScreen/godoap) which will replace the current implementation in Skelerealms. I am debating whether to make GodOAP a dependency or make the GOAP-related stuff in Skelereams a supplementary module.

**Comment by bornato** [comment]
Is the project still alive? It's been over a year since any update has been made. It has some really useful stuff I would really like to try with a short game experiment I want to make.

**Reply by SlashScreen** [reply]
It should be considered still alive! I have just put the project I was writing it for on the backburner. The most recent stuff is on the .7 branch, although that also hasnt updated in a while. It's not abandonware.

**Reply by bornato** [reply]
Okay, I'll try it out. Is there anything to be specially aware of for that last branch?

**Reply by SlashScreen** [reply]
It's been a while so I don't fully remember, but I was in the middle of rewriting a bunch of stuff so the docs may be inaccurate and the savegame might not be working properly. If youre having toruble with something open up a thread here or go on discord
